spring-boot-project/spring-boot-actuator-autoconfigure/src/test/java/org/springframework/boot/actuate/autoconfigure/observation/ObservationAutoConfigurationTests.java

```diff
@@ -129,18 +151,35 @@ void autoConfiguresObservationHandlerWithCustomContext() {
 					ObservationRegistry observationRegistry = context.getBean(ObservationRegistry.class);
 					List<ObservationHandler<?>> handlers = context.getBean(CalledHandlers.class).getCalledHandlers();
 					CustomContext customContext = new CustomContext();
-					Observation.start("test-observation", () -> customContext, observationRegistry);
+					Observation.start("test-observation", () -> customContext, observationRegistry).stop();
```

Comment:
As far as I can tell, this change is unrelated to wrapping the `DefaultMeterObservationHandler` in a `TracingAwareMeterObservationHandler`. If I've got that right, it should be handled separately please.